css/src/components/tabs.scss

```diff
@@ -0,0 +1,110 @@
+$tabs-padding: 0.5em 1.25em !default;
+$tabs-padding-large: 1em 1.5em !default;
+
+$tab-font-color: $primary !default;
+$tabs-hover-background-color: $primary-background !default;
+$tabs-hover-border-color: $primary-hover !default;
+$tabs-active-hover-background-color: $primary-hover !default;
+$tabs-active-hover-border-color: $primary-hover !default;
+$tabs-active-background-color: $primary-active;
+$tabs-active-border-color: $primary-active !default;
+$tabs-active-color: $text-invert !default;
+
+.tabs {
+	display: grid;
+	justify-content: center;
+	width: 100%;
+	grid-gap: 1em;
+	grid-template-columns: max-content auto max-content;
+	white-space: nowrap;
+
+	.tab-list {
+		grid-column: 2;
+		display: flex;
+		align-items: center;
+		justify-content: flex-start;
+		border-radius: $border-radius-rounded;
+		box-shadow: $box-shadow-medium;
+		overflow-x: hidden;
+	}
+
+	.tab-previous,
+	.tab-next {
+		display: flex;
+		align-items: center;
+		padding: $tabs-padding;
+		border: none;
+		border-radius: $border-radius-rounded;
+		background-color: $tabs-active-background-color;
+		color: $tabs-active-color;
+		font-weight: 500;
+		box-shadow: $box-shadow-medium;
+
+		&:hover {
+			border-color: $tabs-active-hover-border-color;
+			background-color: $tabs-active-hover-background-color;
+		}
+	}
+
+	.tab-control {
+		display: flex;
+		position: relative;
+		align-items: center;
+		justify-content: center;
+		padding: $tabs-padding;
+		border: none;
```

Comment:
unsure if this is the correct class for this comment, but these need some "contrast themes" or operating system high contrast attention. we can't tell which is active when that is on. Maybe @BenDMyers can provide a recommendation here?

<img width="542" alt="image" src="https://user-images.githubusercontent.com/30843002/213318951-05aa02e5-c9ca-4986-9a29-003b82ea667e.png">

Reply:
updated, active tab is visible.

Reply:
It's possible changes haven't propagated to the preview link yet, but I'm still not seeing a border in forced colors mode:
<img width="1214" alt="tabs forced colors" src="https://user-images.githubusercontent.com/18060369/213526002-009c9a7f-b893-40b0-8935-4487462557c2.png">

Reply:
Ah, okay. The high contrast theme shows a filled tab successfully.

However, separately, we should ensure the operating system forced colors mode _also_ presents active tab state.

Reply:
updated the borders

Reply:
Here's what the tablist looks like for me in two operating systems forced colors contrast themes:
<img width="826" alt="Tablist in Night Sky contrast theme. Tablist is outlined in white, and tabs are outlined in yellow. The active tab is not indicated." src="https://user-images.githubusercontent.com/18060369/213580821-8051e18a-bfff-4d87-935c-24b2d2276eec.png">
<img width="842" alt="Tablist in Dusk contrast theme. Tablist is outlined in white, and tabs are outlined in blue. The active tab is not indicated." src="https://user-images.githubusercontent.com/18060369/213580823-33ad7ff3-f24f-4bf4-b467-e33528745cfc.png">

In my opinion, the border around the tablist is pretty noisy - I'd recommend removing it.

Also, this isn't differentiating the active tab. To align with the idea of filling the active tab that we do in non-forced colors, I'd recommend applying the following styles to the active tab in forced colors mode:

```css
background-color: SelectedItem;
color: SelectedItemText;
forced-color-adjust: preserve-parent-color;
```

Here's what that looks like in one contrast theme:
<img width="844" alt="Tablist in Night Sky contrast theme. Tablist is not outlined. Tabs are outlined in yellow. The active tab is light blue with black text.." src="https://user-images.githubusercontent.com/18060369/213583093-ea15ba26-a4d3-4343-9468-59e638ab1ca1.png">

Reply:
Minor update: the modern recommendation is `Highlight` instead of `SelectedItem`, and `HighlightText` instead of `SelectedItemText`. My apologies!

Reply:
updated

Reply:
Just verified active tab highlights successfully!